src/tests/index.tsx

```diff
@@ -30,20 +70,29 @@ const Wrapper = ({ children }: { children?: ReactNode }) => {
       >
         <IntlProvider onError={() => {}} defaultLocale={defaultLocale} locale={locale}>
           <LocaleContext.Provider value={{ defaultLocale, locale, setLocale }}>
-            <Router>{children}</Router>
+            <RouterForTests routerConfig={routerConfig}>{children}</RouterForTests>
           </LocaleContext.Provider>
         </IntlProvider>
       </AuthContext.Provider>
     </ApiClientContextController>
   );
 };
 
-function customRender(ui: React.ReactElement, options?: Omit<RenderOptions, 'queries'>): RenderResult;
-function customRender<Q extends Queries>(ui: React.ReactElement, options: RenderOptions<Q>): RenderResult<Q>;
+function customRender(
+  ui: React.ReactElement,
+  options?: Omit<RenderOptions, 'queries'> & ExtraRenderOptions,
+): RenderResult;
+function customRender<Q extends Queries>(
+  ui: React.ReactElement,
+  options: RenderOptions<Q> & ExtraRenderOptions,
+): RenderResult<Q>;
 function customRender<Q extends Queries>(
   ui: React.ReactElement,
-  options?: RenderOptions<Q> | Omit<RenderOptions, 'queries'>,
+  options?: (RenderOptions<Q> | Omit<RenderOptions, 'queries'>) & ExtraRenderOptions,
 ): RenderResult<Q> | RenderResult {
+  function Wrapper({ children }: Pick<WrapperProps, 'children'>) {
+    return <_Wrapper routerConfig={options?.routerConfig}>{children}</_Wrapper>;
+  }
```

Comment:
those could be converted to lambda notation 😁

Reply:
I thought about error's stack trace, I think lambda component won't have display name 🤔 I might be wrong tho.
Anyway getting error in this part of code shouldn't happen too often so I'll make it as a lambda as you suggested